samples/bot-types-of-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,136 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send types of cards using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "01/13/2023 05:00:17 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-types-of-cards-nodejs
+
+---
+# Types of Cards
+
+This sample shows the feature where user can send types of cards using bot.
```

Comment:
This sample shows the feature where user can send different types of cards using bot.

Reply:
fixed